pkg/oc/bootstrap/docker/openshift/admin.go

```diff
@@ -260,3 +265,28 @@ func catFiles(dest string, src ...string) error {
 	}
 	return nil
 }
+
+func getRegistryRoute(list []runtime.Object) *routeapi.Route {
+	var route *routeapi.Route
+	for _, item := range list {
+		if svc, ok := item.(*kapi.Service); ok {
+			svc.Spec.ClusterIP = RegistryServiceIP
+			//construct route obj
+			route = &routeapi.Route{
```

Comment:
we can't add new dependencies on internal api objects.  oc client is going to split out into a repo where it doesn't have access to these types.

Reply:
whats good practice for this? Do we create a local structure representing an object? Any pointers where split was done already?

Reply:
just suggesting you use the external api type instead.